packages/cli/src/utils/string.js

```diff
@@ -0,0 +1,13 @@
+function splitFileFromPath(filePath, extension = 'js') {
```

Comment:
This function suits our purpose well for `scaffold`. I left out considering the extension being passed in for now (ex. `src/triggers/foo.js`)